certification/internal/shell/has_minimal_vulns_unshare.go

```diff
@@ -0,0 +1,75 @@
+package shell
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// HasMinimalVulnerabilitiesCheck evaluates the image to confirm that only vulnerabilities
+// below the Critical or Important severities, using oscap-podman and a corresponding
+// OVAL definition.
+type HasMinimalVulnerabilitiesUnshareCheck struct{}
+
+func (p *HasMinimalVulnerabilitiesUnshareCheck) Validate(image string) (bool, error) {
+	lines, err := p.getDataToValidate(image)
+	if err != nil {
+		log.Debugf("Stdout: %s", lines)
+		return false, fmt.Errorf("%w: %s", errors.ErrImageScanFailed, err)
+	}
+
+	return p.validate(lines)
+}
+
+func (p *HasMinimalVulnerabilitiesUnshareCheck) getDataToValidate(image string) ([]string, error) {
```

Comment:
I know it's not strictly required, but I believe we should start commenting these.